src/Console/PublishCommand.php

```diff
@@ -48,7 +48,7 @@ class PublishCommand extends Command
             'Select' => ['select', 'option', 'options'],
             'Switch' => ['switch'],
             'Table' => ['table', 'columns', 'column', 'rows', 'row', 'cell', 'pagination'],
-            'Tabs' => ['tabs'],
+            'Tabs' => ['tab'],
```

Comment:
Hey @idealerror, thanks for the PR. You will need to include both `tabs` and `tab` due to there being a `tabs` component in the root of the stubs folder:

```suggestion
            'Tabs' => ['tab', 'tabs'],
```

Reply:
Fixed. I didn't see a folder for tabs, missed the blade component.